package.json

```diff
@@ -51,6 +53,7 @@
   "devDependencies": {
     "babel-cli": "^6.7.5",
     "babel-eslint": "^6.0.2",
+    "babel-jest": "^19.0.0",
```

Comment:
Now that we're using `yarn` in CI, this is less important. Without it, however, npm 2 (the default for node v4) would fail to run tests because babel-jest wasn't a sibling of jest.